test/gui/tst_sharing/test.feature

```diff
@@ -308,23 +308,22 @@ Feature: Sharing
         And as "Alice" folder "PARENT" should not exist on the server
         And as "Alice" file "lorem.txt" should not exist on the server
 
-    @skip @issue-9439
+    @issue-9439
     Scenario: sharee deletes a file and folder shared by sharer
         Given user "Alice" has uploaded file with content "ownCloud test text file 0" to "textfile.txt" on the server
         And user "Alice" has created folder "Folder" on the server
         And user "Brian" has been created on the server with default attributes and without skeleton files
         And user "Alice" has shared file "textfile.txt" on the server with user "Brian" with "all" permissions
         And user "Alice" has shared file "Folder" on the server with user "Brian" with "all" permissions
         And user "Brian" has set up a client with default settings
-        When the user waits for file "textfile.txt" to be synced
-        And the user waits for folder "Folder" to be synced
+        When the user waits for the files to sync
         And the user deletes the file "textfile.txt"
```

Comment:
If the step `When the user waits for the files to sync` actually waits for all the files and folders to be synced, should we rephrase the step itself? May be not in this PR, but in a different PR.

Reply:
Yes, we can come with better name and do changes in separate PR